redaxo/src/core/lib/fragment.php

```diff
@@ -98,17 +98,15 @@ public function parse($filename, $delete_whitespaces = true)
 
         $this->filename = $filename;
 
+        ob_start();
         foreach (self::$fragmentDirs as $fragDir) {
             $fragment = $fragDir . $filename;
-            if (is_readable($fragment)) {
-                ob_start();
-                if ($delete_whitespaces) {
-                    preg_replace('/(?:(?<=\>)|(?<=\/\>))(\s+)(?=\<\/?)/', '', require $fragment);
-                } else {
-                    require $fragment;
-                }
-
+            if (@include($fragment)) {
```

Comment:
Hier haben wir wieder das `return`-Problem.
Auch wenn das in Fragmenten vielleicht unwahrscheinlicher ist, dass das genutzt wird, ist trotzdem die Frage, ob es das Wert ist, oder man es hier auch lieber so lässt?